packages/web-react/src/components/Tooltip/TooltipModern.tsx

```diff
@@ -10,9 +10,10 @@ const TooltipModern = (props: TooltipModernProps) => {
   const {
     children,
     enableFlipping: flipProp = true,
+    enableFlippingCrossAxis: flipCrossAxis = true,
+    enableHover = false,
```

Comment:
Shouldn't it be true? Otherwise it would be BC.

Tooltip is currently displayed with hover by default: https://spirit-design-system-storybook.netlify.app/?path=/story/components-tooltip--playground.

Reply:
It is still a draft, so everything could change. When we discussed this issue with others on the team, we all agreed it would be turned off by default. However, there is still time to change it, so we can talk about it more tomorrow in the office.

Reply:
I think that the problem we are solving now is only on the `TooltipModern` component which fully uses the Floating UI. The functionality/component pointed out by @dlouhak will be deprecated, I think.